gammapy/cube/make.py

```diff
@@ -303,10 +332,39 @@ def _make_background(self):
 
         self.maps["background"] = background
 
+    def _make_edisp(self):
+
+        energy_axis = self.geom_true.get_axis_by_name("ENERGY")
+        geom_migra = self.geom_true.to_image().to_cube([self.migra_axis, energy_axis])
```

Comment:
This is a bit annoying, doing a `to_image()` and then a `to_cube()`. But the `EdispMap` class needs the first axis to be `migra`, and the second, `energy`.